CMakeLists.txt

```diff
@@ -160,15 +157,19 @@ if (GIT_SCM AND GITDIR)
 
 else()
     message(STATUS "Not in a git repo")
-  endif()
+    set(GIT_SHA1 "UNKNONW")
+    set(GIT_DATE "UNKNONW")
+    set(GIT_COMMIT_SUBJECT "UNKNONW")
+    set(GIT_BRANCH "UNKNONW")
+endif()
 
-# string(REGEX REPLACE "\#" "" GIT_COMMIT_SUBJECT ${GIT_COMMIT_SUBJECT})
```

Comment:
Any reasons why this is deleted?

`cmake` will shock on commit subjects with `#`

Reply:
The line was in the file twice, one commented out, one used. The used one is still in there.